### ISSUE TITLE
Bug: Variable name "tx" doesn't conform to snake_case naming style(invalid-name)

### ISSUE THREAD

**Issue by islishude** [PyCQA/pylint#2970]
### Steps to reproduce

```python
def tmp_func():
    # Variable name "a" doesn't conform to snake_case naming style
    a = "any"
    print(a)


tmp_func()
```

### Current behavior

Variable name "e" doesn't conform to snake_case naming style
Variable name "tx" doesn't conform to snake_case naming style

### Expected behavior

"e" and "tx" vaiable names confom to style

### pylint --version output

pylint 2.3.1
astroid 2.2.5
Python 3.6.8 (default, May  7 2019, 14:58:50) 
[GCC 5.4.0 20160609]

**Comment by PCManticore** [comment]
@islishude In this case, the linter wants variables with at least length 3. It's not clear from the error message but we this enhancement proposal have https://github.com/PyCQA/pylint/issues/2018 that should clear things up.

**Comment by Lorenzobattistela** [comment]
Is this fixed yet? Is there an option for allowing variables with less than 3 characters?

**Comment by Pierre-Sassoulas** [comment]
No, this is closed because it's a duplicate of #2018 latest specifications are in #7305
